test/aio5gc/context/sessionManagement.go

```diff
@@ -170,9 +183,30 @@ func CreatePDUSession(sessionRequest *nasMessage.PDUSessionEstablishmentRequest,
 }
 
 func ReleasePDUSession(ue *UEContext, pduSessionID int32) (SmContext, error) {
-	return ue.DeleteSmContext(pduSessionID)
```

Comment:
The Context is never released? Did you forget to add the ue.DeleteSmContext in the callback of the event when refactoring?